sdk/storage/Azure.Storage.DataMovement.Files.Shares/CHANGELOG.md

```diff
@@ -3,10 +3,12 @@
 ## 12.3.0-beta.1 (Unreleased)
 
 ### Features Added
```

Comment:
Please also update the changelogs for Azure.Storage.DataMovement and Azure.Storage.DataMovement.Blobs, since we always release all 3 DM Lib packages together.